afl-cmin

```diff
@@ -1,12 +1,19 @@
 #!/usr/bin/env sh
+THISPATH=`dirname ${0}`
+
+# call afl-cmin.py if it can be executed successfully.
+if $THISPATH/afl-cmin.py --help > /dev/null 2>&1; then
+  exec $THISPATH/afl-cmin.py "$@"
+fi
```

Comment:
Should we inform the user about the possiblity of the pyhon script here?
Also, when/why would this ever fail?

Reply:
if dependencies are not installed - or python is not present :)
I don't think we need to inform users IMHO

Reply:
Not sure what would happen with `exec` if python isn't installed?

Reply:
I would just get rid of the awk version and worst case fall back to bash, but we can do that later ;)